### PR TITLE
Soosung Yang

### PR DESCRIPTION
**Pull request by soosung03**

### CODE REVIEW

File: brevets/acp_times.py [hunk 1]
```diff
@@ -41,4 +107,60 @@ def close_time(control_dist_km, brevet_dist_km, brevet_start_time):
...
+    floating_point = 0.0
+
+    if control_dist_km == 0:
+        hours = 1
+        return brevet_start_time.shift(hours=hours)
+    if control_dist_km <= 200:
+        hours = control_dist_km / 15
+    if control_dist_km > 200 and control_dist_km <= 400:
+        hours = control_dist_km / 15
+    if control_dist_km > 400 and control_dist_km <= 600:
```
Comment (erinszabo) [inline brevets/acp_times.py:123]: Since everything under 600km uses the 15 control time, you could have just used one ' if control_dist_km <= 600:'

**Review by erinszabo — COMMENTED** [review]
I think it was a good idea to use tables instead of multiple if statements as we saw in lectures, it's too bad you weren't able to implement that. I also had trouble getting that all working in time and had an inefficient set up. I saw you were working on that from the commented out code.

In tests, you would ideally want just one assert per test. I think you could have gotten away with just one standard case per test if you wanted to test 200, 300, etc. individually, and focus more on edge cases and breaking points.

Overall a good effort for the time and difficulty of the project.

**Comment by soosung03** [comment]
Thank  you for the feedback!
